packages/block-library/src/social-links/edit.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+
+import {
+	InnerBlocks,
+} from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import socialList from '../social-link/social-list';
+
+const ALLOWED_BLOCKS = Object.keys( socialList ).map( ( site ) => {
+	return 'core/social-link-' + site;
+} );
+
+// Template contains the links that show when start
+const TEMPLATE = [
+	[ 'core/social-link-facebook' ],
+	[ 'core/social-link-twitter' ],
+	[ 'core/social-link-instagram' ],
+	[ 'core/social-link-wordpress' ],
+];
+
+export const SocialLinksEdit = function( { className } ) {
+	return (
+		<div className={ className }>
+			<InnerBlocks
+				allowedBlocks={ ALLOWED_BLOCKS }
+				templateLock={ false }
+				template={ TEMPLATE }
```

Comment:
I wonder if the larger button appender might be more useful than four social icons that the user might not even want. It would also solve the issue of the block being empty when unselected.

Reply:
I really like the idea of being a little bit more colorful, opinionated and even playful with this block, by preinserting some blocks. It immediately makes clear what this block is about, moreso than the gray appender, which could be anything, and often is.

However, let's bookmark this idea as a solid alternative for the empty state! Thanks Dan!